src/main/java/com/google/gerrit/extensions/api/changes/Changes.java

```diff
@@ -74,7 +75,7 @@ ChangeApi id(String project, String branch, String id)
     public abstract List<ChangeInfo> get() throws RestApiException;
 
     public QueryRequest withQuery(String query) {
-      this.query = query;
+      this.query = StandardKeyEncoder.encode(query);
```

Comment:
I see two solutions at the moment which would not break backward compatibility:

1. Add an option to `QueryRequest` to enable encoding; disabled by default. This solution is not optimal since `QueryRequest` is an official Gerrit API, which I prefer not to modify.
2. Include the StandardKeyEncoder in this library in order to allow users at least to easily encode their queries by their own. Probably add a facade to hide the `com.google.gwtorm package`. This solution is not optimal since library users probably forget to encode their queries.

What do you think?